src/app/modules/details/test-exec-details.component.ts

```diff
@@ -71,24 +98,20 @@ export class TestExecDetailsComponent implements OnInit, OnDestroy {
   }
 
   async updateDetails(id: string): Promise<void> {
+    this.currentId = undefined;
```

Comment:
why not set the id right away?

Reply:
Because the method first clears all details, then retrieves the new details and fills them in. If the details cannot be retrieved for whatever reason, I figure it would be wrong to set the `currentID`, if the corresponding details haven't actually been retrieved. In particular, it should not be possible to change the log level if the initial retrieval of the log did not succeed.

I'll be factoring out the clearing of the details to make the code, well…, _clearer_. Also, the log level buttons should probably appear disabled if `currentId` is `falsy`.